multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -158,8 +165,11 @@ def __init__(
         self.max_img_num_per_col = max_img_num_per_col
         logger.debug(f"max_img_num_per_col: {max_img_num_per_col}")
 
-        if self.prefix == MMDET_IMAGE:
-            assert mmdet is not None, "Please install MMDetection by: pip install mmdet."
+        if self.prefix == MMDET_IMAGE or self.prefix == MMOCR_TEXT_DET:
```

Comment:
I think we need to figure out a better way here. Otherwise, when you add other tasks in OCR and other CV tasks in the future, this if statement will look very long-winded. Welcome any suggestions here.

Reply:
Maybe we can add a list of these prefixes in the constants.

Reply:
Good idea, we should do this in the next PR.